addons/mail/static/src/activity/activity_menu.scss

```diff
@@ -1,6 +1,8 @@
 .o-mail-activity-menu {
     width: 350px;
     max-height: 400px;
+    min-height: 50px;
+    flex: 0 1 auto;
```

Comment:
Why precise `flex: 0 1 auto`?
That's the initial value, and I don't see any other style rule affecting these styling properties.

Reply:
that's a mistake :)